packages/runtime/runtime-utils/src/dataStoreHelpers.ts

```diff
@@ -47,19 +47,17 @@ export function exceptionToResponse(error: unknown): IResponse {
 	}
 
 	// Capture error objects, not stack itself, as stack retrieval is very expensive operation
-	const errWithStack = generateErrorWithStack();
+	const errWithStack =
+		(typeof error === "object" && error !== null && "stack" in error
```

Comment:
So we think `"stack" in error` will not trigger the string generation?  Makes sense.  Presumably the string generation is in a property getter, which wouldn't be executed for the `in` check.  Cool.

Did you happen to repro the "slow" string generation and observe that it's ok here?  I'm ok either way but curious.